docs/getting-started/index.md

```diff
@@ -1,106 +1,97 @@
+---
+title: Getting started
+---
+
 # Getting Started
 
-Let's get started using Chart.js!
+Let's get started with Chart.js!
 
-First, we need to have a canvas in our page. It's recommended to give the chart its own container for [responsiveness](../configuration/responsive.md).
+* **[Follow a step-by-step guide](./usage) to get up to speed with Chart.js**
+* [Install Chart.js](./installation) from npm or a CDN 
+* [Integrate Chart.js](./integration) with bundlers, loaders, and front-end frameworks
+
+Alternatively, see the example below or check [samples](../samples).
+
+## Create a Chart
+
+In this example, we create a bar chart for a single dataset and render it on an HTML page. Add this code snippet to your page:
 
 ```html
 <div>
   <canvas id="myChart"></canvas>
 </div>
+
+<script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+
+<script>
+  const ctx = document.getElementById('myChart');
+
+  new Chart(ctx, {
+    type: 'bar',
+    data: {
+      labels: ['Red', 'Blue', 'Yellow', 'Green', 'Purple', 'Orange'],
+      datasets: [{
+        label: '# of Votes',
+        data: [12, 19, 3, 5, 2, 3],
+        borderWidth: 1
+      }]
+    },
+    options: {
+      scales: {
+        y: {
+            beginAtZero: true
+        }
+      }
+    }
+  });
+</script>
 ```
 
-Now that we have a canvas we can use, we need to include Chart.js in our page.
+You should get a chart like this:
+
+![example chart](./example-chart.png)
```

Comment:
Keeping the interactive chart over an image would have my preference

Reply:
This is a great idea! IDK why I didn't think about this myself when submitting the PR. I've replaced every screenshot with a live demo (embedded Chart.js chart).